### PR TITLE
Square

### PR DESCRIPTION
**Pull request by peterton**
- new IP lease
- now drive all 3 motors for square

### CODE REVIEW

File: golonomic.go [hunk 1]
```diff
@@ -18,36 +18,52 @@ const (
...
+	if err != nil {
+		log.Fatalf("failed to find large motor on outC: %v", err)
+	}
+	err = motorC.SetStopAction("brake").Err()
+	if err != nil {
+		log.Fatalf("failed to set brake stop for large motor on outC: %v", err)
+	}
 }
 
 func move(x, y, z float64) {
```
Comment (jeffizhungry) [inline golonomic.go]: i don't understand what z is in this case

File: golonomic.go [hunk 2]
```diff
@@ -18,36 +18,52 @@ const (
...
-	fn := mat.Formatted(inverse, mat.Prefix("    "), mat.Squeeze())
-	fmt.Printf("i = %v", fn)
 }
 
-func testMotors() {
-	outA, err := ev3dev.TachoMotorFor("ev3-ports:outA", "lego-ev3-l-motor")
+func setupMotors() {
+	var err error
+
+	motorA, err = ev3dev.TachoMotorFor("ev3-ports:outA", "lego-ev3-l-motor")
```
Comment (jeffizhungry) [inline golonomic.go]: You could probably create a helper function e.g. ```motorA := initMotor("A")```. In this case error is not necessary if you're just going to exit the program within initMotor

File: golonomic.go [hunk 3]
```diff
@@ -18,36 +18,52 @@ const (
 
 var inverse = mat.NewDense(3, 3, nil)
```
Comment (jeffizhungry) [inline golonomic.go]: maybe you can be more specific with the naming, what is the matrix actually used for?

**Review by jeffizhungry — APPROVED** [review]
Approved, but left some comments

File: golonomic.go (hunk 1 above)
Comment (peterton) [reply golonomic.go]: added comment and used better variable name

File: golonomic.go (hunk 2 above)
Comment (peterton) [reply golonomic.go]: done!

File: golonomic.go (hunk 3 above)
Comment (peterton) [reply golonomic.go]: I doubt I can capture that in a variable name so I added a description

**Pull request merged**